source/Cosmos.Common/StringHelper.cs

```diff
@@ -427,6 +427,18 @@ internal enum StringComparisonResultEnum {
             }
             return true;
         }
+
+        /// <summary>
+        /// Check if string lenght is divisible by a given number.
+        /// </summary>
+        /// <param name="aString">A string to check if it's length is divisible.</param>
+        /// <param name="aNum">A denominator</param>
+        /// <returns>Returns TRUE if string length divisible by the given number.</returns>
+        public static bool IsLengthDivisible(string aString, int aNum) {
+            if (aNum != 0 && aString.Length % aNum == 0) {
+                return true;
+            }
+            return false;
```

Comment:
I dont think this has to be its own function

Reply:
I thought it might be useful for the user in the future.
You think its better to remove it completely and do do this operation in the original function, or leave it as helper function but inline the method in the original?

Reply:
Changed it.

Reply:
Can you please remove the method? As it can be easily done inline, I dont think we will need it as a helper

Reply:
Sure